easybuild/easyconfigs/t/TensorFlow-Datasets/TensorFlow-Datasets-4.7.0-foss-2021a-CUDA-11.3.1.eb

```
@@ -0,0 +1,46 @@
+easyblock = 'PythonBundle'
+
+name = 'TensorFlow-Datasets'
+version = '4.7.0'
+versionsuffix = '-CUDA-%(cudaver)s'
+
+homepage = 'https://www.tensorflow.org/datasets'
+description = """
+TensorFlow Datasets is a collection of datasets ready to use, with TensorFlow or other Python ML frameworks, such as
+Jax. All datasets are exposed as tf.data.Datasets , enabling easy-to-use and high-performance input pipelines.
+"""
+
+toolchain = {'name': 'foss', 'version': '2021a'}
+
+dependencies = [
+    ('CUDA', '11.3.1', '', SYSTEM),
+    ('Python', '3.9.5'),
+    ('tqdm', '4.61.2'),
+    ('TensorFlow', '2.6.0', versionsuffix),
+]
+
+sanity_pip_check = True
+use_pip = True
+
+exts_list = [
+    ('promise', '2.3', {
+        'checksums': ['dfd18337c523ba4b6a58801c164c1904a9d4d1b1747c7d5dbf45b693a49d93d0'],
+    }),
+    ('googleapis-common-protos', '1.56.4', {
+        'modulename': 'google',
+        'checksums': ['c25873c47279387cfdcbdafa36149887901d36202cb645a0e4f29686bf6e4417'],
+    }),
+    ('tensorflow-metadata', '1.10.0', {
```

we should use an underscore here, since both the source and python module use `tensorflow_metadata`
```suggestion
    ('tensorflow_metadata', '1.10.0', {
```

---

this fixes the sanity check and source download